addons/api/addon/models/target.js

```diff
@@ -135,6 +135,12 @@ export default class TargetModel extends GeneratedTargetModel {
     return Boolean(pendingOrActiveSessions.length);
   }
 
+  get associatedAliases() {
+    return this.store
+      .peekAll('alias')
```

Comment:
Does this name here refer to the assignment we did in L30, or is it the actual resource name in the store?

Reply:
It's the name of the resources in the stores, so `alias` models